src/WebJobs.Script/Utility.cs

```diff
@@ -792,6 +793,28 @@ public static bool IsMediaTypeOctetOrMultipart(MediaTypeHeaderValue mediaType)
             }
         }
 
+        public static void LogAutorestGeneratedJson(string rootScriptPath, ILogger logger = null)
+        {
+            string autorestGeneratedJsonPath = Path.Combine(rootScriptPath, ScriptConstants.AutorestGeenratedMetadataFileName);
+            JObject autorestGeneratedJson;
+
+            if (FileUtility.FileExists(autorestGeneratedJsonPath))
+            {
+                string autorestGeneratedJsonPathContents = FileUtility.ReadAllText(autorestGeneratedJsonPath);
+
+                try
+                {
+                    autorestGeneratedJson = JObject.Parse(autorestGeneratedJsonPathContents);
+                    logger.AutorestGeneratedFunctionApplication(autorestGeneratedJson.ToString());
+                }
+                catch (JsonException ex)
+                {
```

Comment:
catch any `exception`  to ensure host continues to start if other exceptions happen here.